laborator/content/reprezentare-numere/solution/utils/Makefile.generic

```diff
@@ -0,0 +1,9 @@
+PROGNAME ?= a.out
```

Comment:
All these files are already present in the repo. No need to duplicate them here.